sdk/src/exporter/otprotocol/recordable.cc

```diff
@@ -0,0 +1,28 @@
+#include "src/exporter/otprotocol/recordable.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
```

Comment:
Should exporters live in the internal namespace?

Reply:
I separated out into an exporters folder, similar to how it's done for go and java.